Training/2.0 Libraries.md

```diff
@@ -21,7 +22,11 @@ When to use which? Each of these libraries has very useful functionality but you
 Make a two-dimensional numpy array. First row will be: [1,2,3] and second row will be [4,5,6]. 
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
-```
+```py
+arr = np.array([1,2,3][4,5,6])
+mean = np.mean(arr)
+max = np.max(arr)
+min = np.min(arr)
```

Comment:
Good attempt! To create a multi-row numpy array, you need a nested list with each sub-list (i.e. row of data) separated by commas within the main list: `arr = np.array([[1,2,3],[4,5,6]])`
Min, max and mean are correct, but question asks for indexes of min and max too. Try to work these out, if not I will explain on Friday.

Reply:
Also try Task 2 b) which is slightly trickier, but will be useful to learn.